### ISSUE TITLE
vite-plugin-svelte outputs a11y warnings for `EdgeLabel` 

### ISSUE THREAD

**Issue by mcmxcdev** [xyflow/xyflow#3841]
### Describe the Bug

When building a SvelteKit project having `@xyflow/svelte` installed, these annoying warnings pop up:

```
8:53:24 AM [vite-plugin-svelte] <project>/node_modules/.pnpm/@xyflow+svelte@0.0.35_svelte@4.2.8/node_modules/@xyflow/svelte/dist/lib/components/EdgeLabel/EdgeLabel.svelte:12:2 A11y: visible, non-interactive elements with an on:click event must be accompanied by a keyboard event handler. Consider whether an interactive element such as <button type="button"> or <a> might be more appropriate. See https://svelte.dev/docs/accessibility-warnings#a11y-click-events-have-key-events for more details.
```

```
8:53:24 AM [vite-plugin-svelte] <project>/node_modules/.pnpm/@xyflow+svelte@0.0.35_svelte@4.2.8/node_modules/@xyflow/svelte/dist/lib/components/EdgeLabel/EdgeLabel.svelte:12:2 A11y: <div> with click handler must have an ARIA role
```

### Steps to Reproduce the Bug or Issue

1. Run `vite build` with a SvelteKit project having `@xyflow/svelte` installed
2. Expect warnings to show up

### Expected behavior

No a11y warnings to impact the output log of application build.

### Platform

- OS: Ubuntu 23.10
- Browser: Brave
- Version: 1.62.153

**Comment by moklick** [comment]
thanks! Will be fixed in the next release.

**Comment by mcmxcdev** [comment]
There are still various a11y warnings with the latest `@xyflow/svelte@0.0.36` release:

```
12:21:20 PM [vite-plugin-svelte] /user/node_modules/.pnpm/@xyflow+svelte@0.0.36_svelte@4.2.11/node_modules/@xyflow/svelte/dist/lib/components/EdgeLabel/EdgeLabel.svelte:12:2 A11y: Elements with the 'button' interactive role must have a tabindex value.
12:21:20 PM [vite-plugin-svelte] /user/node_modules/.pnpm/@xyflow+svelte@0.0.36_svelte@4.2.11/node_modules/@xyflow/svelte/dist/lib/components/EdgeLabel/EdgeLabel.svelte:12:2 A11y: visible, non-interactive elements with an on:click event must be accompanied by a keyboard event handler. Consider whether an interactive element such as <button type="button"> or <a> might be more appropriate. See https://svelte.dev/docs/accessibility-warnings#a11y-click-events-have-key-events for more details.
12:21:21 PM [vite-plugin-svelte] /user/node_modules/.pnpm/@xyflow+svelte@0.0.36_svelte@4.2.11/node_modules/@xyflow/svelte/dist/lib/components/NodeSelection/NodeSelection.svelte:22:2 A11y: visible, non-interactive elements with an on:click event must be accompanied by a keyboard event handler. Consider whether an interactive element such as <button type="button"> or <a> might be more appropriate. See https://svelte.dev/docs/accessibility-warnings#a11y-click-events-have-key-events for more details.
12:21:21 PM [vite-plugin-svelte] /user/node_modules/.pnpm/@xyflow+svelte@0.0.36_svelte@4.2.11/node_modules/@xyflow/svelte/dist/lib/components/NodeSelection/NodeSelection.svelte:22:2 A11y: <div> with contextmenu, click handlers must have an ARIA role
```

**Comment by moklick** [comment]
should be fixed in 0.0.39
